src/app/containers/MPulseBeacon/index.jsx

```diff
@@ -23,6 +27,15 @@ const MPulseBeaconContainer = () => {
     }
   }, [isEnabled, personalisationEnabled, API_KEY]);
 
+  useEffect(() => {
+    if (isEnabled && personalisationEnabled) {
+      window.SIMORGH_MPULSE_INFO = {
+        pageType,
+        service,
+      };
+    }
+  }, [isEnabled, personalisationEnabled, pageType, service]);
```

Comment:
As an alternative we could add this info (and perhaps other info such as the app version) to the window regardless so that it's not coupled to our mPulse implementation and can be consumed by other things later.

Reply:
I elected to keep it in MPulse as if we decide to remove mpulse, I didnt want this logic hanging somewhere with no use. Setting onto the window isnt something we should rely on in multiple places if possible. If we find a usecase where we need to use the data elsewhere, IMO we can move it out then :)